$NetBSD$

Allow the package to build on platforms that don't
support libudev and libinput

--- CMakeLists.txt.orig	2019-02-24 00:16:01.000000000 +0000
+++ CMakeLists.txt
@@ -9,10 +9,10 @@ project(lxqt-config)
 set(LXQT_CONFIG_PROJECT "${PROJECT_NAME}")
 
 option(UPDATE_TRANSLATIONS "Update source translation translations/*.ts files" OFF)
-option(WITH_APPEARANCE "Build the 'lxqt-config-appearance'" ON)
+option(WITH_APPEARANCE "Build the 'lxqt-config-appearance'" OFF)
 option(WITH_BRIGHTNESS "Build the 'lxqt-config-brightness'" ON)
 option(WITH_FILE_ASSOCIATIONS "Build the 'lxqt-config-file-associations'" ON)
-option(WITH_INPUT "Build the 'lxqt-config-input'" ON)
+option(WITH_INPUT "Build the 'lxqt-config-input'" OFF)
 option(WITH_LOCALE "Build the 'lxqt-config-locale'" ON)
 option(WITH_MONITOR "Build the 'lxqt-config-monitor'" ON)
 option(WITH_TOUCHPAD "Build touchpad settings" ON)
